docs/deploy/arm_compute_lib.rst

```diff
@@ -26,6 +26,24 @@ and GPU's. Currently the integration offloads operators to ACL to use hand-craft
 routines in the library. By offloading select operators from a relay graph to ACL we can achieve
 a performance boost on such devices.
 
+Installing Arm Compute Library
+------------------------------
+
+We recommend two different ways to build and install ACL:
+
+* Use the script located at `docker/install/ubuntu_install_arm_compute_library.sh`. You can use this
+  script for building ACL from source natively or for cross-compiling the library on an x86 machine.
+  You may need to change the architecture of the device you wish to compile for by altering the
+  `target_arch` variable. Binaries will be built from source and installed to the location denoted by
```

Comment:
Any suggestion/command about how to know the target architecture of the current machine?

Reply:
That's a good point, we could use `lscpu` to display the model name of the CPU, although I don't think there is an easy way to determine the architecture directly.